docs/containers.config.5.md

```diff
@@ -0,0 +1,88 @@
+% containers.conf(5) Container runtime tools configuration file
+
+# NAME
+containers.conf - configuration file of the container runtime tools like podman & buildah
+
+# DESCRIPTION
+The containers configuration file specifies all of the available configuration options and command-line flags for container runtime tools like podman & buildah, but in a TOML format that can be easily modified and versioned.
+
+# FORMAT
+The [TOML format][toml] is used as the encoding of the configuration file. Every option is nested under its table. No bare options are used. The format of TOML can be simplified to:
+
+    [table1]
+    option = value
+
+    [table2]
+    option = value
+
+## CONTAINERS TABLE
+The `containers` table contains settings pertaining to the OCI runtime used and options for how to set up and manage the OCI runtime.
+
+**default_ulimits**=[]
+  A list of ulimits to be set in containers by default, specified as "<ulimit name>=<soft limit>:<hard limit>", for example:"nofile=1024:2048".
+
+**conmon_env**=["PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"]
+  Environment variable list for the conmon process, used for passing necessary environment variables to conmon or the runtime.
+
+**selinux**=false
```

Comment:
Set this to default.

Reply:
If not specified, then it will default to if the System is running in SELinux mode, if system is not running with SELinux, then we should fail.